modular_zubbers/modules/examine/examine_tgui.dm

```diff
@@ -0,0 +1,17 @@
+/datum/examine_panel/ui_data(mob/user)
+	var/list/data = ..()
+	var/datum/preferences/preferences = holder.client?.prefs
+	var/flavor_text_nsfw
+	var/headshot_nsfw
+	var/show_nsfw_flavor_text = preferences?.read_preference(/datum/preference/choiced/show_nsfw_flavor_text)
+	if(issilicon(holder) && !(show_nsfw_flavor_text == "Never"))
```

Comment:
What is the purpose of never showing nsfw ft- at that point you might as well remove it

Reply:
Because you might want to more tightly control when it's visible, such as by changing the preference mid-round to make it visible